ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v3/page/_cq_dialog/.content.xml

```diff
@@ -141,6 +141,38 @@
                                                     sling:resourceType="core/cif/components/renderconditions/pagetype"
                                                     pageType="landing"/>
                                             </myAccountPage>
+                                            <clearCache
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/container">
+                                                <granite:rendercondition
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="core/cif/components/renderconditions/pagetype"
+                                                        pageType="landing"/>
+                                                <granite:rendercondition
```

Comment:
If you check in crx/de these two granite:rendercondition nodes are merged into one node in the repo in an inappropriate way. Each sibling node should have a distinct name. Please use one node only and use the predefined 'and' or 'or' render conditions (like at "magentoSection" further down in this .content.xml) which support multiple render conditions as child nodes and perform a boolean operation on them.